packages/worker/src/worker.ts

```diff
@@ -426,7 +422,7 @@ export class Worker {
     const connection = options.connection ?? (await InternalNativeConnection.connect());
     let nativeWorker: NativeWorkerLike;
     try {
-      nativeWorker = await nativeWorkerCtor.create(connection, compiledOptions);
+      nativeWorker = await nativeWorkerCtor.create(connection, compiledOptions, bundle?.code);
```

Comment:
I had miss that previously, but we should not need to pass code to native worker. Not blocking though.

Reply:
Yeah, I missed this too TBH, it's for generating the buildId, I think we should do this in the worker factory.

Reply:
Refactored this a bit.